src/groupByKey.ts

```diff
@@ -2,6 +2,18 @@ type GroupsMap<T> = {
   [key: string]: T[];
 };
 
-export function groupByKey(items, key) {
-  // write code here;
+export function groupByKey <T>(item: T[], key: keyof T): GroupsMap<T> {
```

Comment:
```suggestion
export function groupByKey <T>(items: T[], key: keyof T): GroupsMap<T> {
```
Actually, it's an array of items, name it correctly